### PR TITLE
Lesson2

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: printNames.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const f = require("fs");
+const emptyfunc =()=>{
+};
+
+const printFunc=(err,files)=>{
+  let myStr = " ";
```
Comment (chocobrownie) [inline printNames.js]: change myStr to a more readable variable

File: printNames.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  files.forEach((e)=> {
```
Comment (chocobrownie) [inline printNames.js]: change e to a more readable variable

File: printNames.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+const emptyfunc =()=>{
+};
+
+const printFunc=(err,files)=>{
+  let othersNames = " ";
+  files.forEach((eachFile)=> {
+    if(eachFile!="saumya")
+      othersNames = othersNames + " " +eachFile;
+  });
+  f.writeFile("/home/sya/garageScript/writeFile.txt",myStr, emptyfunc);
```
Comment (MadhuBeesetty) [inline printNames.js]: saumya you are calling myStr here. where did you use it?

File: printNames.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+const emptyfunc =()=>{
+};
+
+const printFunc=(err,files)=>{
+  let othersNames = " ";
+  files.forEach((eachFile)=> {
+    if(eachFile!="saumya")
+      othersNames = othersNames + " " +eachFile;
+  });
+  f.writeFile("/home/sya/garageScript/writeFile.txt",othersNames, emptyfunc);
```
Comment (MadhuBeesetty) [inline printNames.js]: otherNames is just a variable its not a function. why are you calling it?

Comment (songz) [reply printNames.js]: @MadhuBeesetty it is data that she wants to write into the file

File: writeFile.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+  README alopez amey betty bharath bill cecile colin dsoper emily emmanuel git gitlab-runner hue jia jiarun kinue louis madhu madhub manik mc mike ming olivia pdiddy pete rahul sahil song sya tri yj
```
Comment (songz) [inline writeFile.txt]: please do not commit files that are generated.

File: printNames.js (hunk 4 above)
Comment (songz) [inline printNames.js]: there's no point creating a new variable for an empty function. simply pass in this: `()=>{}`

File: addNum.js [hunk 6]
```diff
@@ -1,7 +0,0 @@
-
```
Comment (songz) [inline addNum.js]: Master should have all your files. I would prefer this to stay in.

**Review by chocobrownie — APPROVED** [review]
13 commits? That's some work there

File: printNames.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+const f = require("fs");
```
Comment (allopez7) [inline printNames.js]: Add a more descriptive name for f. Maybe fs to abbreviate file system.

File: printNames.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+const emptyfunc =()=>{
```
Comment (allopez7) [inline printNames.js]: Optional: you can place this inside the emptyfunc location, just use (err)=>{} or your empty function.

**Review by allopez7 — APPROVED** [review]
Good Job!!!

**Pull request merged**